lib/archethic_web/controllers/faucet_controller.ex

```diff
@@ -57,6 +63,18 @@ defmodule ArchEthicWeb.FaucetController do
         |> put_flash(:error, "Unable to send the transaction")
         |> render("index.html", address: address, link_address: "")
 
+      %{archived?: true, archived_since: archived_since} ->
+        now = System.monotonic_time()
+        archived_elapsed_time = System.convert_time_unit(now - archived_since, :native, :second)
+        archived_elapsed_diff = div(@faucet_rate_limit_expiry, 1000) - archived_elapsed_time
+
+        conn
+        |> put_flash(
+          :error,
+          "Archived address, Try after #{ArchEthic.Utils.seconds_to_human_readable(archived_elapsed_diff)}"
```

Comment:
Updated with block

Reply:
I think we should inform the user with more user friendly sentence because they have requested too much UCO for the timespan.
Also we should add an information in the UI before the submitting form, to explain the rate limit policy

Reply:
UI about the rate limit @samuel-uniris 
![image](https://user-images.githubusercontent.com/9107477/160993588-84c6b3cf-ed9f-4be6-8098-a69b35fcc5aa.png)